### DISCUSSION TITLE
Missing track section

### DISCUSSION THREAD

**Discussion by vulturo26** [rickloader/NewForestRouteV3#14] [Q&A] [answered]
When I attempt to load a route (I've tried three or four) I get a message that "Open Rails detected a track section which is not present in tsection.dat ... Most likely you don't have the XTracks or YTracks version needed..."

I'm not computer savvy. Can you please tell me in simple terms how to put this right? Thanks

**Comment by rickloader** [comment, marked as answer]
Routes often need a particular track system which is not included in the route download. You have to add the track system to the "global" folder. OR is complaining that the Route is asking for track you don`t have. ( NewForest Route does contain the UKFS track system).
Look in the Route readme to find the track system required.
Join trainsim.com and look for the tracks in the file library. Down load and install in the Global folder
For example x-tracks https://www.trainsim.com/vbts/tslib.php?searchid=22149666
Rick

**Comment by vulturo26** [comment]
Thanks!

On Tue, 27 Sept 2022 at 13:23, rickloader ***@***.***> wrote:

> Routes often need a particular track system which is not included in the
> route download. You have to add the track system to the "global" folder. OR
> is complaining that the Route is asking for track you don`t have. (
> NewForest Route does contain the UKFS track system).
> Look in the Route readme to find the track system required.
> Join trainsim.com and look for the tracks in the file library. Down load
> and install in the Global folder
> For example x-tracks
> https://www.trainsim.com/vbts/tslib.php?searchid=22149666
> Rick
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/rickloader/NewForestRouteV3/discussions/14#discussioncomment-3742981>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/A3BAWJ2XBDJA4Z6H5FJC633WALRLHANCNFSM6AAAAAAQWUCWW4>
> .
> You are receiving this because you authored the thread.Message ID:
> ***@***.***
> com>
>


-- 
Stephen Butler
3 Golf Links Court
Tadcaster
LS24 9JD
N. Yorkshire
